### ISSUE TITLE
does not affect LAST_REPEAT

### ISSUE THREAD

**Issue by aspiers** [progfolio/epoch#3]
Just now I used `epoch-agenda-todo` on this entry to mark that I did some yoga yesterday:

```org
******* NEXT [#A] practice yoga
        SCHEDULED: <2020-10-09 Fri 07:00 .+2d/3d>
        :PROPERTIES:
        :STYLE:    habit
        :LAST_REPEAT: [2020-10-07 Wed 07:21]
        :END:
        :LOGBOOK:
        - State "DONE"       from "NEXT"       [2020-10-07 Wed 07:15]
        - State "DONE"       from "NEXT"       [2020-10-01 Thu 18:45]
        :END:
```

The result was as follows:

```org
******* NEXT [#A] practice yoga
        SCHEDULED: <2020-10-12 Mon 07:00 .+2d/3d>
        :PROPERTIES:
        :STYLE:    habit
        :LAST_REPEAT: [2020-10-10 Sat 12:02]
        :END:
        :LOGBOOK:
        - State "DONE"       from "NEXT"       [2020-10-09 Fri 07:00]
        - State "DONE"       from "NEXT"       [2020-10-07 Wed 07:15]
        - State "DONE"       from "NEXT"       [2020-10-01 Thu 18:45]
        :END:
```

As you can see, while the `SCHEDULED` and `LOGBOOK` sections were updated correctly, `LAST_REPEAT` was not - I would expect it to be set to `[2020-10-09 Fri 07:00]`.

**Comment by progfolio** [comment]
Sorry, this was a known issue I just hadn't had the time to look at. I think I've fixed it on latest master. Let me know if that works for you after an upgrade.

**Comment by aspiers** [comment]
Seems to work perfectly now - thanks a lot!  Just for the record, the fix was e7bc042d593aff6235bd6a996eef2f47ca10f5d7
